src/components/atoms/AccessDeniedModal/AccessDeniedModal.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback, useState } from "react";
+import { Button, Modal } from "react-bootstrap";
+import { useHistory } from "react-router-dom";
+import { venueLandingUrl } from "utils/url";
+
+interface AccessDeniedModalProps {
+  show?: boolean;
+  venueId: string;
+  venueName: string;
+}
+
+export const AccessDeniedModal: React.FC<AccessDeniedModalProps> = ({
+  show = true,
+  venueId,
+  venueName,
+}) => {
+  const [isVisible, setIsVisible] = useState(show);
```

Comment:
Since the `show` prop is passed into `useState` it will be used as the initial value, but never be changed after that. So if the prop is changed externally, the modal isn't going to respond to that.

Reply:
That's the whole point. This is just a "default value"

Reply:
As a dev passing props to a component, I expect the component to change based on those props. This breaks that expectation.

If the show prob is intended to only ever be a default here, then it should be named something like defaultShow or similar to make that explicit/obvious.

Reply:
[NB] It seems we never resolved this part.